src/shared/meson.build

```diff
@@ -332,6 +333,7 @@ libshared_name = 'systemd-shared-@0@'.format(shared_lib_tag)
 
 libshared_deps = [threads,
                   libacl,
+                  libaudit,
```

Comment:
Now libshared depends on libaudit.
Then, maybe we can drop the dependencies on it at src/core, src/sysusers, and src/sysupdate ?

Reply:
I get linker errors if I try to remove the dependencies there, so it doesn't seem so